src/Components/BlockWithList/BlockWithList.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { BaseSyntheticEvent, useState } from 'react'
+import { withKnobs, text } from '@storybook/addon-knobs'
+
+import BlockWithList from './index'
+import Button from '../OpiumButton'
+
+import { ETheme, sectionThemes } from '../../Constants/Types/theme.types'
+
+export const dropdownSelector = () => {
+  const items = [
```

Comment:
Add props with data. We should get data from outside

Reply:
So we have 2 props (frontSide and backSide). "const items = [..." is no needed.